aztk/spark/client.py

```diff
@@ -20,7 +21,14 @@ def __init__(self, secrets_config):
     '''
     def create_cluster(self, cluster_conf: models.ClusterConfiguration, wait: bool = False):
         try:
-            zip_resource_files = upload_node_scripts.zip_scripts(self.blob_client, cluster_conf.custom_scripts, cluster_conf.spark_configuration)
+            if cluster_conf.user_configuration:
+                user_conf = yaml.dump({'username': cluster_conf.user_configuration.username,
+                                       'password': cluster_conf.user_configuration.password,
```

Comment:
We may want to omit password until we support encryption.

Reply:
If we disable password here that means that we would no longer support adding a user with password at cluster creation time. That's a potentially breaking change, so maybe this should PR should wait for or include the encryption feature.

Reply:
If the goal is just to not have the password in cleartext, we could also hash it, and create the user using the hash. 

That would break ssh'ing outside of AZTK, though.